db/models.py

```diff
@@ -50,3 +55,52 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        formatted_date = self.created_at.strftime("%Y-%m-%d %H:%M:%S")
+        return f"{formatted_date}"
+
+    class Meta:
+        ordering = ['-created_at']
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey("MovieSession", on_delete=models.CASCADE)
+    order = models.ForeignKey("Order", on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    def __str__(self) -> str:
+        return f"{self.movie_session.movie.title} {self.movie_session.show_time.strftime('%Y-%m-%d %H:%M:%S')} (row: {self.row}, seat: {self.seat})"
+
+    def clean(self) -> None:
+        if self.row > self.movie_session.cinema_hall.rows:
+            raise ValidationError(
+                {"row": f"row number must be in available range: (1, rows): (1, {self.movie_session.cinema_hall.rows})"})
+        if self.seat > self.movie_session.cinema_hall.seats_in_row:
+            raise ValidationError(
+                {"seat": f"seat number must be in available range: (1, seats_in_row): (1, {self.movie_session.cinema_hall.seats_in_row})"})
+
+    def save(self, *args: Any, **kwargs: Any) -> None:
+        self.full_clean()
+        return super().save(*args, **kwargs)
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(fields=["movie_session", "row", "seat"], name="unique_movie_session"),
```

Comment:
The `UniqueConstraint` name 'unique_movie_session' is too generic. Consider renaming it to something more descriptive, such as 'unique_ticket_per_session_row_seat', to avoid potential conflicts with other constraints.